server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -79,6 +80,7 @@ export class ScriptoriumLambda implements IPartitionLambda {
 				this.pendingMetric = Lumberjack.newLumberMetric(
 					LumberEventName.ScriptoriumProcessBatch,
 					{
+						metricCorrelationId: uuid(),
```

Comment:
Each metric already has an internal property called id
https://github.com/microsoft/FluidFramework/blob/c79eff6afdbc4d74c3c7ab9b375d57082c732ef8/server/routerlicious/packages/services-telemetry/src/lumber.ts#L33

In our logs we can verify those through the `lumberId` column. Maybe we can use that here to correlate the metrics?

Reply:
Hey Henrique, I believe lumberId is different for each metric, right? I want to correlate ScriptoriumProcessBatch metric logs with RunWithRetry metric logs.. can we do that with any existing properties?

Reply:
For context - for every ScriptoriumProcessBatch, there can be multiple RunWtihRetry logs and RunWithRetry has a property for documentId, but ScriptoriumProcessBatch doesnt because it can contain a batch of multiple docIds. So if I am investigating a particular docId and its RunWithRetry logs, I need to find its corresponding ScriptoriumProcessBatch log.. Currently I do that by checking the logs around that timestamp but I think a correlation Id would be more helpful, unless we already have something like that?

Reply:
Hi @shubhi1092, thanks for the clarifications! Understood. Correct, each metric has a different `lumberId`.

From the above, let me check if my understanding is correct: we have multiple `RunWithRetry` metrics/logs, and we need to correlate those with one `ScriptoriumProcessBatch` metric/log. Would this work? Fetching the 
`id` property of the `ScriptoriumProcessBatch` `Lumber` metric and using that as one of the telemetry parameters given to `RunWithRetry()`?

Reply:
Updated.